packages/@aws-cdk/aws-codedeploy/lambda-packages/ecs-deployment-provider/lib/on-event.ts

```diff
@@ -0,0 +1,116 @@
+import { Logger } from '@aws-lambda-powertools/logger';
+import { CodeDeploy } from '@aws-sdk/client-codedeploy';
+
+const logger = new Logger({ serviceName: 'ecsDeploymentProviderOnEvent' });
+const codedeployClient = new CodeDeploy({});
+
+/**
+ * The properties of the CodeDeploy Deployment to create
+ */
+interface DeploymentProperties {
+  description: string;
```

Comment:
Can you add docstrings for all the properties in this file.

Reply:
Done